app/assets/javascripts/mobile/mobile_file_uploader.js

```diff
@@ -2,15 +2,14 @@
 //= require js_image_paths
 
 function createUploader(){
-
    var aspectIds = gon.preloads.aspect_ids;
 
    new qq.FileUploaderBasic({
        element: document.getElementById('file-upload-publisher'),
        params: {'photo' : {'pending' : 'true', 'aspect_ids' : aspectIds},},
        allowedExtensions: ['jpg', 'jpeg', 'png', 'gif', 'tiff'],
        action: "/photos",
-       debug: true,
+       debug: false,
```

Comment:
Not needed I guess, according to https://github.com/Valums-File-Uploader/file-uploader/blob/master/client/fileuploader.js#L270

It works with debug set to false on Firefox and Chromium at least.